packages/language/test/fourslash-harness/harness-interface.ts

```diff
@@ -15,8 +22,19 @@ export interface HarnessTesterInterface {
     expectLinks(): void;
     /**
      * Expect that the defined links do not link to the given label.
+     * @param label The label to expect no links at.
      */
-    expectNoLinksAt(label: string | number): void;
+    expectNoLinksAt(label: Label): void;
+  };
+
+  completion: {
+    /**
+     * Expect that the completion items at the given label contains the given content.
+     *
+     * @param label The label to expect the completion items at.
+     * @param content The completion items to expect.
+     */
+    expectAt(label: Label, content: string[]): void;
```

Comment:
Suggestion: Can we use the `{ includes: string[], excludes: string[] }` object here instead? We used to have this, right?